### DISCUSSION TITLE
E: What are the potential specific benefits?

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#8] [Ideas]
This thread is for discussing, finalizing and organizing a breakout with the theme in the title above. 

Original text: 

># Where are the greatest benefits to be had? And to whom? 

- What are the incentives to get started?
- Which stage of the research workflow should initial efforts be targeted at?
    - Hypothesising/expt. design
    - Data acquisition
    - Processing and analysis
    - Dissemination
    - Archiving
- What are the possible futures/user stories this could enable? For example:
    - Current researchers wanting to collaborate with others using different tools and data management systems, and publish linked data instead of raw archives
    - Future researchers trying to assess and reproduce published research, and perform metastudies, machine learning or knowledge representation
    - Software/method/tool developers, who can support a well-defined interface without reinventing the wheel, and can broadcast what they support to the community
- Interoperability approaches “biased” by requiring a critical mass on both sides of a mapping, to the exclusion of long-tail, smaller projects

**Comment by kjappelbaum** [comment]
Can we collect a set of *concrete* user stories about what interoperability could enable --- for different users (experimental chemists, data scientists,. ...): 

- From my own domain: Linking crystal structures and isotherms (https://chemrxiv.org/engage/chemrxiv/article-details/61c300059efae7891a230138) would allow ML studies, benchmarking of force fields, assess reproducibility, ... 
- If we do something like the [Compound Platform](https://www.ioc.kit.edu/braese/1020.php) every sample URI could lead to an "order form" to get a sample of the original compound. That is, we can link the physical and digital world -- should drastically enhance reusability and reproducibility  


What do we learn about the design of an infrastructure based on the user stories? What technology is actually needed? 

Quoting [Raymond](https://en.wikipedia.org/wiki/The_Cathedral_and_the_Bazaar): To solve an interesting problem, start by finding a problem that is interesting to you.

**Comment by kjappelbaum** [comment]
At some point, I was looking for some examples of how people used semantic technologies to arrive at some new insights.

Does anyone know some?

**Comment by sphuber** [comment]
> What are the incentives to get started

One major problem that is holding back the quality of data management is the fact there currently is almost _no_ incentive to do so. At least in academia, there is almost no remunerative system to reward scientists that produce high quality data sets. This is slowly starting to change, but I really think that if we can induce a cultural shift where data set citations become as valuable (or similar) as publication citations, scientists would be much more inclined to spend the effort to curate their data and metadata. Part of the solution is already there, with a variety of data repositories being online that provide long-term storage for data sets with persistent identifiers. Perhaps we should talk to publishers to have them put more focus on data sets being cited like any other publication and bringing that more to the forefront. This is clearly a very long term problem, but valuable nonetheless.

**Reply by kjappelbaum** [reply]
Indeed, I think as soon as Google Scholar would index data set and code citations things would change ...

**Reply by ml-evs** [reply]
Do you think we also need to move beyond bibliometrics? I would personally find it valuable to publish linked data (either datasets, or just all bits and bobs) if I knew there was the infrastructure to ingest it, explore it etc. I can't say I've found e.g. [Google Dataset Search](https://datasetsearch.research.google.com/) very useful yet.

If anything, I worry that we are currently at a stage where it is very easy to make your data open and citable in an unhelpful way, e.g. unstructured archives of raw spectral data, excel spreadsheets etc. Initiatives like the new RSC Digital Discovery journal's [code/data reviewers](https://blogs.rsc.org/dd/2021/12/16/data-reviewer/?doing_wp_cron=1639653260.2619609832763671875000) could push back, but I imagine the technical publishing guidelines for chemists/matsci data will not be particularly strict... Perhaps this could be a focus of our perspective paper, I wonder even if Digital Discovery would be interested in hosting it.

**Reply by kjappelbaum** [reply]
>Do you think we also need to move beyond bibliometrics? 

For sure, however, this is what people care about atm and I think that there will always be an incentive to measure something. 
The good thing about citations is that, if done properly, they will give more rewards to those datasets that can be reused. 

I guess at the moment only a few actually reuse experimental data. Hence it is a bit of a chicken and egg problem. How do you convince people to do something for which you have no nice use cases (or other benefits).

**Reply by kjappelbaum** [reply]
>I wonder even if Digital Discovery would be interested in hosting it.

We might even consider asking Digital Discovery if they want to promote the workshop. However, this might make us too "dependent" on them

**Comment by kjappelbaum** [comment]
perhaps the practically most relevant question:
- what are the 20% that give us 80% of the cake? (open, structured data?) what % is linking, what % is using URIs for everything,  REST-API with `openapi.json`...
